gum/arch-arm64/gumarm64writer.c

```diff
@@ -872,6 +870,25 @@ gum_arm64_writer_put_load_store_pair_post (GumArm64Writer * self,
       (rt2 << 10) | (rn << 5) | rt);
 }
 
+static void
+gum_arm64_writer_put_load_store_pair_signed_offset (
+    GumArm64Writer * self,
+    GumArm64MemPairOperandSize op_size,
+    guint opc,
+    gboolean v,
+    gboolean l,
+    guint rt,
+    guint rt2,
+    guint rn,
+    gssize signed_offset)
+{
+  gsize shift = gum_mem_pair_offset_shift (op_size, v);
```

Comment:
I tend to avoid declaration and assignment on the same line (unless it's just assigning a constant), and especially when the surrounding code is a bit “noisy”. So to make it more readable I would suggest:

```C
{
  gsize shift;

  shift = gum_mem_pair_offset_shift (op_size, v);

  gum_arm64_writer_put_instruction (self, (opc << 30) | (5 << 27) |
  ...
```